Libplanet/Net/NetMQTransport.cs

```diff
@@ -328,6 +359,20 @@ public async Task RunAsync(CancellationToken cancellationToken)
             cancellationToken
         );
 
+        /// <summary>
+        /// Adds given <paramref name="peers"/> to routing table by sending <see cref="Ping"/>.
+        /// </summary>
+        /// <param name="peers"><see cref="IEnumerable{Peer}"/> of peers to add.</param>
```

Comment:
How about this? If it is to represent it is plural, the word `peers` will present it enoughly. Or if it is to represent its type, it can be represented in document or code level.

```suggestion
        /// <param name="peers">The peers to add.</param>
```